### PR TITLE
Shared infra schema Doumentation

### PR DESCRIPTION
**Pull request by AlexBrad1eyCT**
Movement of entities per #122 plus schema documentation

### CODE REVIEW

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Documentation.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+The _IfcSharedInfrastructureElements_ schema defines the shared or common elements (subtypes of _IfcElement_) that are applicable to the infrastructure sector.
+
+the scope of this includes concepts to cover the definition of:
+
+* Geology & terrain
```
Comment (SergejMuhic) [inline IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Documentation.md:5]: Not sure about geology and earthworks.

**Review by SergejMuhic — APPROVED** [review]
Looks good. After @larswik let us merge.

File: IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Documentation.md (hunk 1 above)
Comment (AlexBrad1eyCT) [reply IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Documentation.md]: well the earthworks entities are in this package, i changed geology to geotechnical

Comment (SergejMuhic) [reply IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Documentation.md]: Yes, of course. Let us not start any discussion on this now. Treat my comment as an observation for traceability.

**Review by larswik — APPROVED** [review]

**Pull request merged**